src/cortxfs/cortxfs_ops.c

```diff
@@ -183,6 +203,25 @@ int cfs_access(struct cfs_fs *cfs_fs, const cfs_cred_t *cred,
 	return rc;
 }
 
+int cfs_access(struct cfs_fs *cfs_fs, const cfs_cred_t *cred,
+	       const cfs_ino_t *ino, int flags)
+{
+	size_t rc;
+#ifdef ENABLE_TSDB_ADDB
+	perfc_trace_inii(PFT_CFS_ACCESS, PEM_CFS_TO_NFS);
```

Comment:
`perfc_trace_ini`?

Reply:
Went through utils changes and saw that this has been there so it isn't a mistake and hence resolving this comment.

Reply:
perfc_trace_ini is a separate API, please refer c-utils/src/include/perf/perf-counters.h, perfc_trace_ini v/s perfc_trace_inii